README.md

```diff
@@ -97,7 +97,7 @@ It's preferable to use content in this collection using their Fully Qualified Co
         msg: "ID is {{ my_droplet.data.droplet.id }}, IP is {{ my_droplet.data.ip_address }}"
 
     - name: Tag a resource; creating the tag if it does not exist
-      digital_ocean_tag:
+      tag:
```

Comment:
This won't work. Either use `community.digitalocean.tag` or `digital_ocean_tag`.